packages/dashboard/lib/prisma.ts

```diff
@@ -1,11 +1,8 @@
 import { PrismaClient } from '@prisma/client';
 
-// @ts-expect-error NodeJS.Global does not exists
-interface CustomNodeJsGlobal extends NodeJS.Global {
+declare const global: typeof globalThis & {
```

Comment:
Smart! Could you duplicate the same inside `lib/redis.ts` and `lib/s3.ts`?

Reply:
Done !